include/IECore/MeshAlgo.h

```diff
@@ -47,6 +47,23 @@ namespace IECore
 namespace MeshAlgo
 {
 
+class TexturePrimVarNames
+{
+	public:
+		TexturePrimVarNames( const std::string &positionPrimVarName, const std::string &uvSetName );
+
+		std::string positionPrimVarName;
+		std::string uvSetName;
+
+		std::string uName() const;
+		std::string vName() const;
+
+		std::string tangentName() const;
+		std::string binormalName() const;
+
+		std::string indicesName() const;
```

Comment:
Likewise for the indices, these won't be stored separately in Cortex 10. I was hoping that introducing MeshAlgo would start moving us in the right direction for Cortex 10 but without having to make the compatibility break yet, so I do think this is a backwards step.